NGTrackForce/src/app/components/client-list/client-list.component.ts

```diff
@@ -74,6 +74,7 @@ export class ClientListComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
+    this.lsHelp.removeStorageItem("clientGetAll");
```

Comment:
why do this?

Reply:
local window storage needs global helper